### DISCUSSION TITLE
Alias notes and description cannot be saved or updated.

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#569] [Self-hosting question] [answered]
When I create a generate a new Random Alias, any Notes or Description entered and click save, the error below shows in red box in browser, no errors in any docker logs or in browser console that I can see.

Unknown Error
Sorry for the inconvenience! Could you refresh the page & retry please?

If I create a new Custom Alias and add a desription on this screen and save it, the note IS saved.
But cannot be edited after creation, the error above is displayed.

If I connect via mobile app and try edit a Note, it says;
Unknown Error, network connection was lost.

Again I dont see in any logs about this, almost like the request doesnt get sent, or something is stopping it being recieved, am I meant to be opening additional ports or nginx routing for this to work?

With everything else working flawlessly, Im pretty confused about this.
Any pointers in where to look for whats going wrong?

**Comment by nguyenkims** [comment, marked as answer]
Do you know on what frontend this issue happens: is it the browser extension, the android, iOS or web app?

And do you have the latest version of these clients?

**Reply by ghost** [reply]
Ok. Sorry I've just clicked as to what I've done...

After finding suspect access attempt in nginx logs, I went searching for an explanation and prevention and came across something like;
add_header Allow "GET, POST, HEAD" always; 
if ( $request_method !~ ^(GET|POST|HEAD)$ ) 
{ return 444; }

Now little did ol nooby me know, PUT was being used when editing the alias field.. I was dropping my own connnection.

Sorry....

**Reply by nguyenkims** [reply]
No worries, glad that you found the cause!
